internal/bazel_integration_test/git_repository_under_test.bzl

```diff
@@ -69,6 +69,41 @@ _attrs = {
         default = False,
         doc = "Whether to clone submodules in the repository.",
     ),
+    "patch_args": attr.string_list(
```

Comment:
can't this be a `**kwargs` to repository_rule or whatever you're wrapping, so we don't have to declare/document so many attributes?

Reply:
moved to https://github.com/bazelbuild/rules_nodejs/pull/1105; in this case I am duplicating the implementation of the git_repository repository rule here but I use the vendored utils.bzl for most of the work so its mostly a shell implementation so have to define these attributes here